### PR TITLE
Delete SwirldState1.

### LINKED ISSUE #5453: Clean up SS1 code
We currently do not maintain the SS1 codebase. Delete these code pathways.

### PR DESCRIPTION
**Pull request by cody-littley**
Closes #5453

### CODE REVIEW

**Review by lpetrovic05 — APPROVED** [review]

File: platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/state/SwirldStateManagerSingle.java
Comment (alittley) [inline platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/state/SwirldStateManagerSingle.java:871]: With this deletion, log marker `QUEUES` has no remaining usages

**Review by alittley — DISMISSED** [review]

File: platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/state/SwirldStateManagerImpl.java
Comment (alittley) [inline platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/state/SwirldStateManagerImpl.java:48]: Theoretically with the renaming of this class, the new `PlatformThreads` drawio diagram which just merged into develop would need to be updated to match.

File: platform-sdk/swirlds-common/src/main/java/com/swirlds/common/system/SwirldState.java
Comment (alittley) [inline platform-sdk/swirlds-common/src/main/java/com/swirlds/common/system/SwirldState.java:75]: I notice now that you deleted the parameter docs. Was that intentional?

File: platform-sdk/platform-apps/tests/PlatformTestingTool/src/main/java/com/swirlds/demo/platform/PlatformTestingToolState.java
Comment (alittley) [inline platform-sdk/platform-apps/tests/PlatformTestingTool/src/main/java/com/swirlds/demo/platform/PlatformTestingToolState.java:1581]: Should the value of `initialized` be checked here, like in `handleConsensusRound`?

**Review by alittley — DISMISSED** [review]

File: platform-sdk/swirlds-common/src/main/java/com/swirlds/common/system/SwirldState.java
Comment (cody-littley) [reply platform-sdk/swirlds-common/src/main/java/com/swirlds/common/system/SwirldState.java:75]: Added parameter javadocs

File: platform-sdk/platform-apps/tests/PlatformTestingTool/src/main/java/com/swirlds/demo/platform/PlatformTestingToolState.java
Comment (cody-littley) [reply platform-sdk/platform-apps/tests/PlatformTestingTool/src/main/java/com/swirlds/demo/platform/PlatformTestingToolState.java:1581]: It was not previously being checked. This implementation is a direct copy-paste from `PlatformTestingState2.java`, which was collapsed into this file.

My general approach for PTT is to avoid changing business logic at all costs, since if I break something I have to spend time debugging it. This is especially true when the deletion of this testing app is on the horizon.

Are you ok leaving this the way it is for now?

Comment (alittley) [reply platform-sdk/platform-apps/tests/PlatformTestingTool/src/main/java/com/swirlds/demo/platform/PlatformTestingToolState.java:1581]: Sounds good to me

**Review by alittley — DISMISSED** [review]

File: platform-sdk/swirlds-common/src/main/java/com/swirlds/common/system/SwirldState.java
Comment (poulok) [inline platform-sdk/swirlds-common/src/main/java/com/swirlds/common/system/SwirldState.java:25]: Why are these comments deleted?

File: platform-sdk/swirlds-common/src/main/java/com/swirlds/common/system/SwirldState.java
Comment (poulok) [inline platform-sdk/swirlds-common/src/main/java/com/swirlds/common/system/SwirldState.java:62]: Some documentation for `preHandle()` was lost here. Check what was on the `SwirldState2.prehandle()` method before

File: platform-sdk/swirlds-common/src/main/java/com/swirlds/common/system/SwirldState.java
Comment (poulok) [inline platform-sdk/swirlds-common/src/main/java/com/swirlds/common/system/SwirldState.java:74]: Lots of documentation was lost here

**Review by poulok — CHANGES_REQUESTED** [review]
Please ensure documentation is not lost on the `SwirldState` methods

File: platform-sdk/swirlds-common/src/main/java/com/swirlds/common/system/SwirldState.java
Comment (cody-littley) [reply platform-sdk/swirlds-common/src/main/java/com/swirlds/common/system/SwirldState.java:25]: IMO, existing javadocs were over-fitted to the hedera/PTT use case. Also, a lot of the advice is out of date (e.g. telling the app how to synchronize methods. Let's discuss.

**Review by poulok — DISMISSED** [review]
Approved, pending approval from Leemon on the swirld state documentation

**Review by alittley — DISMISSED** [review]

File: platform-sdk/swirlds-common/src/main/java/com/swirlds/common/system/SwirldState.java
Comment (cody-littley) [reply platform-sdk/swirlds-common/src/main/java/com/swirlds/common/system/SwirldState.java:25]: this has been reverted

File: platform-sdk/swirlds-common/src/main/java/com/swirlds/common/system/SwirldState.java
Comment (cody-littley) [reply platform-sdk/swirlds-common/src/main/java/com/swirlds/common/system/SwirldState.java]: this has been reverted

**Review by lpetrovic05 — APPROVED** [review]

File: platform-sdk/swirlds-common/src/main/java/com/swirlds/common/system/SwirldState.java
Comment (alittley) [reply platform-sdk/swirlds-common/src/main/java/com/swirlds/common/system/SwirldState.java:83]: @cody-littley Inherit doc not relevant

**Review by alittley — APPROVED** [review]

**Review by edward-swirldslabs — APPROVED** [review]

**Review by jeromy-cannon — APPROVED** [review]

**Review by qnswirlds — APPROVED** [review]

**Pull request merged**
